app/controllers/boards_controller.rb

```diff
@@ -38,6 +39,6 @@ def update
 
   def board_params
     params.require(:board)
-          .permit(:title, :heroku_api_key, :app_name, :repository_url)
+          .permit(:title, :heroku_api_key, :app_name)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 6) spaces for indenting an expression spanning multiple lines.